### ISSUE TITLE
playerProfile calls returning 500

### ISSUE THREAD

**Issue by brandly** [nickb1080/nba#27]
```js
nba.stats.playerProfile({ 
  PlayerID: 201939
}).then(profile => {
  // do something
}, error => {
  console.log(error)
})
```

i'm seeing an error like

```
[Error: 500 Internal Server Error – {"Message":"An error has occurred."}]
```

i'm guessing the API changed underneath you, but i haven't done any digging yet.

thanks nick!

**Comment by nickb1080** [comment]
😞 Yea I noticed this too – it's causing the test suite to fail. I'll take a spin through stats.nba.com and see if I can find them doing a valid request. Usually the error messages are pretty descriptive (they'll tell you which field is missing or malformed) but this one is a head-scratcher.

**Comment by brandly** [comment]
ha! yeah, it doesn't give you much to work with.

i just clicked [around stats.nba.com a bit](http://stats.nba.com/player/#!/203076/career/), and found a request to `http://stats.nba.com/stats/playerprofilev2?LeagueID=00&PerMode=PerGame&PlayerID=203076`

since they're using a `v2` url now, i'm guess this change affects [`nba-client-template`](https://github.com/nickb1080/nba-client-template) instead of this repo

hope this helps! ⭐️

**Comment by nickb1080** [comment]
nice find -- thanks!!

**Comment by nickb1080** [comment]
fixed & published in 3.1.1 -- @brandly thanks for the assist 🏀 

(travis seems to be flaking out on something unrelated -- the actual tests are passing)
